recipes/iterativewgcna/meta.yaml

```diff
@@ -21,7 +21,8 @@ requirements:
   run:
     - matplotlib
     - python
-    - rpy2
+    - rpy2  # [not win]
+    - rpy2 <3.0  # [win]
```

Comment:
Sorry it's not immediately obvious to me. Why is this needed?

Reply:
From [this thread](https://bitbucket.org/rpy2/rpy2/issues/577/windows-conda-rpy2-version) and my own experience, rpy2 >= 3.0 doesn't support Windows fully yet. Hopefully this will be fixed in the future.